test/unit/error.test.ts

```diff
@@ -424,4 +397,147 @@ describe('MongoErrors', () => {
       });
     });
   });
+
+  describe('retryable errors', () => {
+    describe('#isRetryableWriteError', () => {
+      const tests: {
+        description: string;
+        result: boolean;
+        error: MongoError;
+        maxWireVersion: number;
+      }[] = [
+        {
+          description: 'a plain error',
+          result: false,
+          // @ts-expect-error: passing in a plain error to test false case
+          error: new Error('do not retry me!'),
+          maxWireVersion: 2.3
+        },
+        {
+          description: 'a MongoError with no code nor label',
+          result: false,
+          error: new MongoError('do not retry me!'),
+          maxWireVersion: 2.3
+        },
+        {
+          description: 'a MongoWriteConcernError with no code nor label',
+          result: false,
+          error: new MongoWriteConcernError({ message: 'empty wc error' }),
+          maxWireVersion: 2.3
+        },
+        {
+          description: 'a MongoWriteConcernError with a random label',
+          result: false,
+          error: new MongoWriteConcernError(
+            { message: 'random label' },
+            { errorLabels: ['myLabel'] }
+          ),
+          maxWireVersion: 0
+        },
+        {
+          description: 'a MongoWriteConcernError with an ExceededTimeLimit code above server 4.4',
+          result: false,
+          error: new MongoWriteConcernError({}, { code: 262 }),
+          maxWireVersion: 9
+        },
+        {
+          description: 'a MongoWriteConcernError with an ExceededTimeLimit code below server 4.4',
```

Comment:
If ExceededTimeLimit code is supposed to represent a retryable write code, the test description should reflect that instead of calling attention to a specific label

Reply:
Fixed